src/xdocs/checks/annotation/annotationusestyle.xml

```diff
@@ -130,116 +130,136 @@
-  &lt;property name=&quot;closingParens&quot; value=&quot;always&quot;/&gt;
-  &lt;property name=&quot;trailingArrayComma&quot; value=&quot;ignore&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AnnotationUseStyle&quot;&gt;
+      &lt;property name=&quot;elementStyle&quot; value=&quot;compact&quot;/&gt;
+      &lt;property name=&quot;closingParens&quot; value=&quot;always&quot;/&gt;
+      &lt;property name=&quot;trailingArrayComma&quot; value=&quot;ignore&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
 &lt;/module&gt;
         </source>
-        <p>
-          Example:
-        </p>
+        <p id="Example3-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;unchecked&quot;) // OK
-@Deprecated // Violation - must have a closing parenthesis
-@SomeArrays({&quot;unchecked&quot;,&quot;unused&quot;}) // OK
-public class TestOne
+@SuppressWarnings(&quot;unchecked&quot;) // ok as element style set to compact
+@Deprecated // violation 'Annotation must have closing parenthesis'
+@SomeArraysUnique({&quot;unchecked&quot;,&quot;unused&quot;}) // ok as it's in compact style
+public class Example3
 {
 
 }
 
-@SuppressWarnings(value={&quot;unchecked&quot;}) // Violation - parameter 'value' shouldn't be used
-@Deprecated() // OK
-@SomeArrays(value={&quot;unchecked&quot;,&quot;unused&quot;,}) // Violation - parameter 'value' shouldn't be used
-class TestTwo {
+// violation below 'Annotation style must be 'COMPACT''
+@SuppressWarnings(value={&quot;unchecked&quot;})
+@Deprecated() // ok as closingParens set to always
+// violation below 'Annotation style must be 'COMPACT''
+@SomeArraysUnique(value={&quot;unchecked&quot;,&quot;unused&quot;,})
+class TestStyle3 {
 
 }
         </source>
-        <p>
+
+        <p id="Example4-config">
           To configure the check to enforce a trailing array comma,
-            with ignoring the elementStyle and a closing parenthesis.
+          with ignoring the elementStyle and a closing parenthesis.
         </p>
         <source>
-&lt;module name=&quot;AnnotationUseStyle&quot;&gt;
-  &lt;property name=&quot;elementStyle&quot; value=&quot;ignore&quot;/&gt;
-  &lt;property name=&quot;closingParens&quot; value=&quot;ignore&quot;/&gt;
-  &lt;property name=&quot;trailingArrayComma&quot; value=&quot;always&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AnnotationUseStyle&quot;&gt;
+      &lt;property name=&quot;elementStyle&quot; value=&quot;ignore&quot;/&gt;
+      &lt;property name=&quot;closingParens&quot; value=&quot;ignore&quot;/&gt;
+      &lt;property name=&quot;trailingArrayComma&quot; value=&quot;always&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
 &lt;/module&gt;
         </source>
-        <p>
-          Example:
-        </p>
+        <p id="Example4-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;unchecked&quot;) // OK
```

Comment:
Restore this list annotation

Reply:
done. thank you for poinitng it out. i somehow missed both the annotation.